script.js

```diff
@@ -0,0 +1,578 @@
+  setCapsLock() {
+    const arrKeys = this.elements.keyBox.querySelectorAll('.key:not(.key--function)');
+
+    for (let i = 0; i < arrKeys.length; i += 1) {
+      const elem = arrKeys[i];
+      const keySymbol = elem.innerHTML.toUpperCase();
+      elem.innerHTML = keySymbol;
+    }
+  },
+
+
+  // method switch Language Layout for keyboard, argument:(not)
+  switchLanguageLayout() {
+    if (this.inputAria.languageToggle === 'en') {
+      this.inputAria.languageToggle = 'ru';
+    } else if (this.inputAria.languageToggle === 'ru') {
+      this.inputAria.languageToggle = 'en';
+    }
+    this.setLanguageLayout(this.inputAria.languageToggle);
+  },
+
+
+  // method type from screen keys
+  type(scr) {
+    const text = this.elements.textAria.value;
+    const st = this.elements.textAria.selectionStart;
+    const end = this.elements.textAria.selectionEnd;
+
+    this.elements.textAria.value = `${text.substring(0, st)}${scr}${text.substring(end)}`;
+    this.elements.textAria.selectionStart = st + 1;
+    this.elements.textAria.selectionEnd = st + 1;
+  },
+
+
+  // method type Backspace key
+  typeBackspace() {
+    const text = this.elements.textAria.value;
+    const st = this.elements.textAria.selectionStart;
+    const end = this.elements.textAria.selectionEnd;
+
+    if (st === end) {
+      this.elements.textAria.value = text.substring(0, st - 1) + text.substring(end);
+      this.elements.textAria.selectionStart = st - 1;
+      this.elements.textAria.selectionEnd = st - 1;
+    } else {
+      this.elements.textAria.value = text.substring(0, st) + text.substring(end);
+      this.elements.textAria.selectionStart = st;
+      this.elements.textAria.selectionEnd = st;
+    }
+    this.elements.textAria.focus();
+  },
+
+
+  // method delete symbol in textAria
+  typeDelete() {
+    const text = this.elements.textAria.value;
+    const st = this.elements.textAria.selectionStart;
+    const end = this.elements.textAria.selectionEnd;
+
+    if (st === end) {
+      this.elements.textAria.value = text.substring(0, st) + text.substring(end + 1);
+      this.elements.textAria.selectionStart = st;
+      this.elements.textAria.selectionEnd = st;
+    } else {
+      this.elements.textAria.value = text.substring(0, st) + text.substring(end);
+      this.elements.textAria.selectionStart = st;
+      this.elements.textAria.selectionEnd = st;
+    }
+    this.elements.textAria.focus();
+  },
+
+
+  // method toggle CapsLock
+  toggleCapsLock() {
+    const elm = this.elements.keyBox.querySelector('button[data-key-code="CapsLock"]');
+    if (!this.inputAria.capsLockToggle) {
+      elm.classList.add('turn-on');
+      this.setCapsLock();
+      this.inputAria.capsLockToggle = true;
+    } else {
+      elm.classList.remove('turn-on');
+      this.setShiftUpLayout();
+      this.inputAria.capsLockToggle = false;
+    }
+  },
+
+
+  // method handle FunctionKeys
+  handleFunctionKeys(keyCode) {
+    switch (keyCode) {
+      case 'BackSpace':
```

Comment:
I think it is better to store such strings into a constant variable

Reply:
where the best place is constant variables in my case? I have one big object KeyBoard.

Reply:
Done